sim/src/NA6PVerTel.cxx

```diff
@@ -107,23 +119,31 @@ void NA6PVerTel::createGeometry(TGeoVolume* world)
   auto* pixelStationVol = new TGeoVolume("PixelStationVol", pixelStationShape, NA6PTGeoHelper::instance().getMedium(addName("Air")));
   TGeoVolume* pixelSensor = new TGeoVolume("PixelSensor", sensorShape, NA6PTGeoHelper::instance().getMedium(addName("Silicon")));
   pixelSensor->SetLineColor(NA6PTGeoHelper::instance().getMediumColor(addName("Silicon")));
+  auto* carbonplateShape = new TGeoBBox("CarbonPlateShape", carbonPlateDX / 2, carbonPlateDY / 2, carbonPlateDz / 2);
+  TGeoVolume* cbPlate = new TGeoVolume("CarbonPlate", carbonplateShape, NA6PTGeoHelper::instance().getMedium(addName("CarbonFiber")));
+  cbPlate->SetLineColor(NA6PTGeoHelper::instance().getMediumColor(addName("CarbonFiber")));
 
   // place sensors to station
   std::vector<float> alpdx{pixChipDX / 2 + pixChipOffsX, -pixChipDX / 2 + pixChipOffsY, -pixChipDX / 2 - pixChipOffsX, pixChipDX / 2 - pixChipOffsY};
   std::vector<float> alpdy{pixChipDY / 2 - pixChipOffsY, pixChipDY / 2 + pixChipOffsX, -pixChipDY / 2 + pixChipOffsY, -pixChipDY / 2 - pixChipOffsX};
   for (size_t ii = 0; ii < alpdx.size(); ++ii) {
     auto* sensorTransform = new TGeoTranslation(alpdx[ii], alpdy[ii], 0);
     pixelStationVol->AddNode(pixelSensor, composeSensorVolID(ii), sensorTransform);
+    int nns = pixelStationVol->GetNodes()->GetEntries() - 1;
+    auto* cbTransform = new TGeoTranslation(alpdx[ii], alpdy[ii], pixChipDz / 2 + carbonPlateDz / 2);
+    pixelStationVol->AddNode(cbPlate, composeNonSensorVolID(ii + 40), cbTransform);
```

Comment:
Don't we have a single carbon plate for the whole station? Here you are adding 1 plate per chip, i.e. 4 separate plates of the same size as the chip is.